client/app/core/dialog-utilities.service.js

```diff
@@ -0,0 +1,39 @@
+export function DialogUtilitiesFactory() {
+  const service = {
+    validateField: validateField,
+    validateAllFields: validateAllFields,
+  };
+
+  return service;
+
+  function validateField(field) {
+    const fieldValue = field.default_value;
+    if (field.required) {
+      if (fieldValue === '') {
+        field.fieldValidation = false;
+        field.errorMessage = __('field is required');
```

Comment:
Leading with lowercase doesn't conform to patternfly validation example presented [here](http://www.patternfly.org/pattern-library/forms-and-controls/errors-and-validation/)